### PR TITLE
[DW] boards added

### PR DESCRIPTION
**Pull request by DamianWdowiak**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -5,7 +5,9 @@
   namespace :api, defaults: { format: :json } do
     namespace :v1, defaults: { format: :json } do
       resources :firsts, only: %i[index]
-      resources :workspaces
+      resources :workspaces do
+        resources :boards, only: %i[index show create update destroy]
```
Comment (MateuszRoslak) [inline config/routes.rb]: Only is redundant if all actions are present. AFAIK

File: db/migrate/20211111121403_create_boards.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class CreateBoards < ActiveRecord::Migration[6.1]
+  def change
+    create_table :boards do |t|
+      t.string :name, null: false
+      t.integer :visibility, default: 0
```
Comment (MateuszRoslak) [inline db/migrate/20211111121403_create_boards.rb:5]: Maybe consider a boolean field?

**Review by MateuszRoslak — APPROVED** [review]

File: db/migrate/20211111121403_create_boards.rb (hunk 2 above)
Comment (DamianWdowiak) [reply db/migrate/20211111121403_create_boards.rb:5]: 3 options on boolean? Good luck, it's not gonna be used for now anyways.

Comment (MateuszRoslak) [reply db/migrate/20211111121403_create_boards.rb:5]: Thought it will be only 2 options. So maybe an enum?

**Pull request merged**